Makefile

```diff
@@ -139,13 +144,113 @@ help-install:
 #
 ###############################################################
 
+### Deployment ################################################
+#
+# General release steps:
+# --------------------------------------------------------------
+#  (1) commit all files and remove any untracked files
+#      `git status` should show nothing.
+#  (2) `make release-changelog` to print a suggested
+#      update to CHANGELOG.md. Replace "XXX" with your
+#      intended tag.
+#  (3) create an annotated tag (with -a or -s)
```

Comment:
tiny nit: is there a way to slurp in the changelog into the annotated tag's log?  not a big deal if it's not easy.

Reply:
I'll give it a try.

Reply:
Added `make release-tag TAG=0.0.0`